CHANGELOG.md

```diff
@@ -7,9 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.11.0 - 04-01-2025
+
+### Added
+
+* YAML support for `validate-inputs` command ([#79](https://github.com/stjude-rust-labs/sprocket/pull/79)).
+* Extend `explain` to display related rules, list tags using `--t`, show WDL definitions using `--definitions` ([#80](https://github.com/stjude-rust-labs/sprocket/pull/80)).
+
 ### Changed
 
-* By deafult, when checking a local file, suppress diagnostics from remote files. Added a `--show-remote-diagnostics`
+* Updated WDL crates to latest ([#79](https://github.com/stjude-rust-labs/sprocket/pull/79)). This added many features and fixes. Some highlights:
+    * Fixed certain misplaced highlights from the `ShellCheck` lint.
+    * Relaxed the `CommentWhitespace` lint rule so it doesn't trigger for as many comments.
+    * The `ImportSort` lint rule now supplies the correct order of imports in the `fix` message.
```

Comment:
We've generally avoided having an additional level in the lists. I don't think it's necessarily bad, just pointing out the deviation.